Cargo.lock

```diff
@@ -3880,18 +3880,18 @@ checksum = "28988d872ab76095a6e6ac88d99b54fd267702734fd7ffe610ca27f533ddb95a"
 
 [[package]]
 name = "openssl-src"
-version = "300.0.2+3.0.0"
+version = "111.17.0+1.1.1m"
```

Comment:
I was wondering what was going on here, and it seems to be due to performance regressions in OpenSSL 3.0.0: https://github.com/sfackler/rust-openssl/pull/1578